src/vivarium_gates_bep/components/artifact_builder.py

```diff
@@ -68,6 +69,39 @@ def write_demographic_data(artifact, location):
     write(artifact, key, load(key))
 
 
+def compute_aggregate_prevalence_weighted_disability_weights(disease, loader):
+    def check_mutual_exclusivity(subclass_prevalence, parent_prevalence):
+        sc_sum = sum([df.mean().mean() for df in subclass_prevalence.values()])
```

Comment:
is the default axis zero?

Reply:
Yes, df.mean() gets you this:
draw_0      0.002246
draw_1      0.002003
draw_2      0.002274
draw_3      0.002032
...